types/victory/index.d.ts

```diff
@@ -1068,9 +1068,9 @@ declare module 'victory' {
          * The animate prop should also be used to specify enter and exit
          * transition configurations with the `onExit` and `onEnter` namespaces respectively.
          * @example
-         * {duration: 500, onExit: () => {}, onEnter: {duration: 500, before: () => ({y: 0})})}
+         * {duration: 500, onExit: () => {}, onEnter: {duration: 500, before: () => ({y: 0})}}
          */
-        animate?: AnimatePropTypeInterface;
+        animate?: boolean | AnimatePropTypeInterface;
```

Comment:
Are both `true` and `false` possible?  If not, consider using a literal type to more clearly document the expected behavior.